.github/workflows/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -x
+
+maker=$1
+device=$2
+
+mydir=$(dirname $0)
+source ${mydir}/setup_env.sh
+
+print "======================================== Build"
+make -j8 || exit 10
+
+print "======================================== Install"
+make -j8 install || exit 11
+ls -R ${top}/install
+
+print "======================================== Verify build"
+ldd_result=$(ldd test/tester) || exit 12
+echo "${ldd_result}"
+
+# Verify that tester linked with cublas or rocblas as intended.
+if [ "${device}" = "gpu_nvidia" ]; then
```

Comment:
I think it is missing one = sign. Usually I use `==` for strings.

Reply:
POSIX uses one `=` sign. BASH allows `==` with the same meaning. See https://stackoverflow.com/questions/20449543/shell-equality-operators-eq and:
```
man [
...
     s1 = s2       True if the strings s1 and s2 are identical.
```